clients/ops/privacy-center/pages/consent.tsx

```diff
@@ -146,9 +146,10 @@ const Consent: NextPage = () => {
       }
     );
     (await response.json()) as ApiUserConcents;
+    router.push("/");
```

Comment:
imo might be nice to have a toast message too

Reply:
A toast would look nice but I'm not sure how it would work after the page redirect. If a full page refresh happens the toast probably wouldn't persist. I suppose the redirect could happen after the toast disappears. Let's merge this and see what people think once they start using it